docs/installation.md

```diff
@@ -101,25 +104,25 @@ $ go install
 ### Docker Hub
 Replace [YOUR_CACHE_DIR] with the cache directory on your machine.
 
-```
+```bash
 $ docker pull aquasec/trivy
```

Comment:
This will pull the latest, so if I'm reading docs for 0.16.0, but 0.17.0 is the latest this command won't pull what I expect?

Reply:
Nice catch! https://github.com/aquasecurity/trivy/commit/d17f570d494ab2f68fea0b8c53ae0fab82a63135